app/jobs/event/approval_cleanup_job.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Pfadibewegung Schweiz. This file is part of
+#  hitobito_pbs and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_pbs.
+
+# Clean the text-comments to approvals made for event-applications
+class Event::ApprovalCleanupJob < RecurringJob
+  run_every 1.day
+
+  def perform_internal
+    clean_approval_comments(approval_ids)
+  end
+
+  def cutoff_date = 3.months.ago
+
+  def approval_ids
+    approvals.pluck(:id)
+  end
+
+  def clean_approval_comments(ids) # rubocop:disable Metrics/MethodLength
+    return if ids.blank?
+
+    Event::Approval
+      .where(id: ids)
+      .update_all( # rubocop:disable Rails/SkipsModelValidations
+        comment: nil,
+        current_occupation: nil,
+        current_level: nil,
+        occupation_assessment: nil,
+        strong_points: nil,
+        weak_points: nil
+      )
+  end
+
+  private
+
+  # TODO: move this into Event::Date?
```

Comment:
Remove this comment.

I would also suggest moving this into Event::Date, since it may be useful in upcoming features. It would probably also make sense to get the `cutoff_date` as a parameter :rocket:

Reply:
I've decided to leave it in, for future extraction. The method can be moved, but I couldn't test it properly only in the wagon. 

The cutoff-date is only used in this wagon, so a configurable setting just moves this one file away without any benefit. If this were in the core, the date would surely become a setting.